lightly/cli/config/lightly-serve.yaml

```diff
@@ -0,0 +1,10 @@
+
+input_dir: ''     # Path to the input directory.
+lightly_dir: ''   # Path to the lightly directory.
+host: 'localhost' # Hostname for serving the data.
+port:             # Port for serving the data.
+
+
+hydra:
+  run:
+    dir: lightly_outputs/${now:%Y-%m-%d}/${now:%H-%M-%S}
```

Comment:
What does this do?

Reply:
Hydra creates output directories for storing logs etc. This changes the default location to `lightly_outputs/date/time` to be consistent with the other configuration files we use.

Reply:
Do we need them?

Reply:
I removed them.